coralproject/talk
# Comment Status Fixes

## What does this PR do?

If a comment is posted, then approved, then edited, the status on the comment should be reset to before it was approved. Rejected comments should stay rejected.

Related: https://www.pivotaltracker.com/story/show/150381206

## How do I test this PR?

1. Create a comment in PREMOD.
2. Approve it.
3. Edit it while the edit window is still open.
4. Notice it go back to PREMOD.
